data-raw/grid_nwfscSurvey.R

```diff
@@ -1,18 +1,24 @@
 path <- fs::path(tempdir(), "california_current_grid.rda")
 url <- "https://github.com/James-Thorson-NOAA/FishStatsUtils/raw/main/data/california_current_grid.rda"
 
-download.file(
+downloader::download(
   url = url,
   destfile = path
 )
+
 load(path)
 unlink(path)
 
-california_current_grid <- california_current_grid %>%
-  dplyr::rowwise() %>%
+california_current_grid <- california_current_grid |>
+  dplyr::rowwise() |>
   dplyr::mutate(
     pass_scaled = 0,
     vessel_year = 0,
+    region = 0,
+    split_mendocino = dplyr::case_when(Lat > 40.1666667 ~"N", .default = "S"),
+    split_conception = dplyr::case_when(Lat > 34.45 ~"N", .default = "S"),
+    split_monterey = dplyr::case_when(Lat > 36.0 ~"N", .default = "S"),
+    split_state = dplyr::case_when(Lat > 46.0 ~"W", Lat < 42.0 ~"C", .default = "O"),
```

Comment:
The Washington/Oregon border should be 46.25, not 46.0 ("Beginning one marine league at sea off the mouth of the Columbia river at north latitude 46° 15'00".00": https://app.leg.wa.gov/rcw/default.aspx?cite=43.58.060)

Reply:
fixed: 328a079deb131cfa96451a44a126d9d4bfb3023b